packages/wrangler/src/init.ts

```diff
@@ -942,25 +930,33 @@ async function getWorkerConfig(
 			(durableObject) => (durableObject as { class_name: string }).class_name
 		);
 
-	const routeOrRoutes = routes.map((rawRoute) => {
-		const { id: _id, ...route } = rawRoute;
-		if (Object.keys(route).length === 1) {
-			return route.pattern;
-		} else {
-			return route as Route;
-		}
-	});
-	const routeOrRoutesToConfig =
-		routeOrRoutes.length > 1
-			? { routes: routeOrRoutes }
-			: { route: routeOrRoutes[0] };
+	const allRoutes: Route[] = [
+		...routes.map(
+			(r) => ({ pattern: r.pattern, zone_name: r.zone_name } as ZoneNameRoute)
+		),
+		...customDomains.map(
+			(c) =>
+				({
+					pattern: c.hostname,
+					zone_name: c.zone_name,
+					custom_domain: true,
+				} as CustomDomainRoute)
+		),
+	];
 
 	return {
+		name: workerName,
+		main: entrypoint,
+		workers_dev: workersDev.enabled,
```

Comment:
An undefined `workers_dev` property is inferred as either true or false depending on the value of `routes`, and so I thought it would be better to be explicit